support/rally.js

```diff
@@ -179,3 +179,8 @@ module.exports = class Rally {
     }
   }
 }
+
+// Make this library Require-able.
```

Comment:
I believe you can change your output format from `iife` to `umd` and then you don't need to do this.

Reply:
Thanks, as we talked on Matrix we won't be always using a bundler (e.g. tests) so that won't help :(